planseqlearn/psl/kitchen_mp_env.py

```diff
@@ -324,21 +486,21 @@ def set_robot_based_on_ee_pos(
             progress_thresh=20.0,
             max_steps=1000,
         )
-        if is_grasped:
-            self.sim.data.qpos[7:9] = gripper_qpos
-            self.sim.data.qvel[7:9] = gripper_qvel
+        # if is_grasped:
+        #     self.sim.data.qpos[7:9] = gripper_qpos
+        #     self.sim.data.qvel[7:9] = gripper_qvel
 
-            # compute the transform between the old and new eef poses
-            ee_old_mat = pose2mat((old_eef_xpos, old_eef_xquat))
-            ee_new_mat = pose2mat((self._eef_xpos, self._eef_xquat))
-            transform = ee_new_mat @ np.linalg.inv(ee_old_mat)
+        #     # compute the transform between the old and new eef poses
+        #     ee_old_mat = pose2mat((old_eef_xpos, old_eef_xquat))
+        #     ee_new_mat = pose2mat((self._eef_xpos, self._eef_xquat))
+        #     transform = ee_new_mat @ np.linalg.inv(ee_old_mat)
 
-            # apply the transform to the object
-            new_object_pose = mat2pose(
-                np.dot(transform, pose2mat((object_pos, object_quat)))
-            )
+        #     # apply the transform to the object
+        #     new_object_pose = mat2pose(
+        #         np.dot(transform, pose2mat((object_pos, object_quat)))
+        #     )
 
-        if open_gripper_on_tp:
+        if not is_grasped:
             self.sim.data.qpos[7:9] = np.array([0.04, 0.04])
             self.sim.data.qvel[7:9] = np.zeros(2)
             self.sim.forward()
```

Comment:
why was this code commented out? don't you want to transform the object pose if its grasped?

Reply:
for kitchen + mopa, there's never a case where you teleport with an object in hand so i removed it, but i guess i could put it back for completeness

Reply:
yeah I would leave it there for completeness